src/rtctools_heat_network/heat_mixin.py

```diff
@@ -4466,22 +4812,36 @@ def history(self, ensemble_member):
         return history
 
     def goal_programming_options(self):
+        """
+        Here we set the goal programming configuration. We use soft constraints for consecutive
+        goals.
+        """
         options = super().goal_programming_options()
         options["keep_soft_constraints"] = True
         return options
 
     def solver_options(self):
+        """
+        Here we define the solver options. By default we use the open-source solver cbc and casadi
+        solver qpsol.
+        """
         options = super().solver_options()
         options["casadi_solver"] = "qpsol"
         options["solver"] = "cbc"
```

Comment:
We should change the default to highs

Reply:
this is done in absolute heat merge request